tarantool-lsp/websocket-lib.lua

```diff
@@ -21,7 +18,9 @@ local create_handler = function(options)
     log.info('Path: %s', path)
 
     if not initalized then
-        local ok, err = docs:init({ completions_dir = fio.pathjoin(path, 'completions') })
+        local ok, err = docs:init({
```

Comment:
Why you have two separate pieces of init code?

Reply:
I mean, do you have an ability to use default handlers from `tarantool-lsp/methods.lua`?

Reply:
For support use other completions

Reply:
But by default it will use default handlers

Reply:
You can use the default handler with different options passed into the handler arguments